frame/collective/src/benchmarking.rs

```diff
@@ -35,18 +37,17 @@ use frame_system::Module as System;
 use crate::Module as Collective;
 
 const SEED: u32 = 0;
-
 const MAX_BYTES: u32 = 1_024;
 
-fn assert_last_event<T: Config<I>, I: Instance>(generic_event: <T as Config<I>>::Event) {
+fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::Event) {
 	let events = System::<T>::events();
 	let system_event: <T as frame_system::Config>::Event = generic_event.into();
 	// compare to the last event record
 	let EventRecord { event, .. } = &events[events.len() - 1];
 	assert_eq!(event, &system_event);
 }
 
-benchmarks_instance! {
```

Comment:
why did you change this to benchmark ?

NOTE: I think pallet-balance should make use of benchmarks_instance.

Reply:
ok it is not supported, waiting on https://github.com/paritytech/substrate/pull/8190